house_of_refuge/frontend/src/scripts/index.js

```diff
@@ -34,28 +39,37 @@ const CoordinatorsHeader = ({coordinators, helped, hide}) => {
     <div className="coordinators">
       <div className="d-flex justify-content-around">
         <div className={"mx-5 text-center"}>
-          <h5>Koordynatorzy Zachodni</h5>
+          <h5>{t('terrain_coordinators')}</h5>
           <ol>{(coordinators.station || []).map(c => <li key={c.user.id}>{c.user.display}</li>)}</ol>
         </div>
         <div className={"mx-5 text-center"}>
-          <h5>Koordynatorzy Zdalni</h5>
+          <h5>{t('remote_coordinators')}</h5>
           <ol>{(coordinators.remote || []).map(c => <li key={c.user.id}>{c.user.display}</li>)}</ol>
         </div>
       </div>
       {peopleHelped ?
-          <div><h5 className="good-message">Pomogliśmy
-            dziś {peopleHelped} osobom {"🙏".repeat(Math.floor(peopleHelped / 10))}</h5>
+          <div><h5 className="good-message">
+            {t('today_we_helped_count', {pplHelpedCount: peopleHelped})} {"🙏".repeat(Math.floor(peopleHelped / 10))}</h5>
           </div> : <></>}
     </div>
-    <div><a href="/accounts/logout">wyloguj się</a></div>
+    <div>
+      {Object.keys(lngs).map((lng) => (
```

Comment:
we either can remove this switch from the backoffice for now or use the component from master